src/openms/source/FORMAT/ConsensusXMLFile.cpp

```diff
@@ -88,7 +86,11 @@ namespace OpenMS
     }
     else if (tag == "IdentificationRun")
     {
-      consensus_map_->getProteinIdentifications().push_back(prot_id_);
+      // post processing of ProteinGroups (hack)
```

Comment:
maybe quickly say what the hack consists of

Reply:
haha I dont know. it was copied from idXML